Whisky/View Models/BottleVM.swift

```diff
@@ -59,4 +59,15 @@ class BottleVM: ObservableObject {
             }
         }
     }
+
+    func validateBottleName(bottleName: String) -> (isInvalidName: Bool, description: String) {
```

Comment:
Wouldn’t it be easier to read if the returned bool was isValidName?

Reply:
Indeed you're right. It's a better practice to check for truthfulness. I'll change the logic 👍